src/renderers/dom/client/syntheticEvents/__tests__/SyntheticKeyboardEvent-test.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+var SyntheticKeyboardEvent;
+var getEventCharCode;
+
+describe('SyntheticKeyboardEvent', function() {
+  var createEvent;
+
+  beforeEach(function() {
+    // Mock getEventCharCode for proper unit testing
+    jest.mock('getEventCharCode');
+    getEventCharCode = require('getEventCharCode');
+
+    SyntheticKeyboardEvent = require('SyntheticKeyboardEvent');
+
+    createEvent = function(nativeEvent) {
+      var target = require('getEventTarget')(nativeEvent);
+      return SyntheticKeyboardEvent.getPooled({}, '', nativeEvent, target);
+    };
+  });
+
+  describe('KeyboardEvent interface', function() {
+    describe('charCode', function() {
+      describe('when event is `keypress`', function() {
+        it('returns whatever getEventCharCode returns', function() {
+          getEventCharCode.mockReturnValue(100500);
+          var keyboardEvent = createEvent({type: 'keypress', charCode: 50});
+
+          expect(keyboardEvent.charCode).toBe(100500);
+        });
+      });
+
+      describe('when event is not `keypress`', function() {
+        it('returns 0', function() {
+          var keyboardEvent = createEvent({type: 'keyup', charCode: 50});
+
```

Comment:
Tiny nit: let’s remove line breaks where there are just two lines in the test.
Thanks!